package.json

```diff
@@ -159,7 +159,7 @@
     "regenerator-runtime": "^0.13.2",
     "stylelint-config-terra": "^3.0.0",
     "stylelint": "^10.0.1",
-    "terra-toolkit": "^5.2.0",
+    "terra-toolkit": "cerner/terra-toolkit#master",
```

Comment:
Can we update this to the latest release now?

Reply:
yes!